universal-login-sdk/test/helpers/SetupSdk.ts

```diff
@@ -4,11 +4,11 @@ import UniversalLoginSDK from '@universal-login/sdk';
 import {providers} from 'ethers';
 
 declare interface SetupSdkOverrides {
-  givenProvider?: providers.Provider;
-  overridePort?: number;
+  givenProvider?: providers.Web3Provider;
+  overridePort?: string;
```

Comment:
Looks very similar to `Overrides`, why not merge it into one type in commons, like ProviderConfig?